about.html

```diff
@@ -0,0 +1,281 @@
+                             <div class="card-content">
+                                 <h3 class="name">Mike Tyson</h3>
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+            </article>
+            
+        </article>
+    </section>
+
+    <section class="footer-container">
+        <footer class="footer">
+            <article class="footer-top-container">
+                <!-- <article class="container footer-top">
+                    <h2>Our Partners</h2>
+                    <ul class="partner-logo">
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
+                    </ul>
+                </article> -->
+            </article>
+          <article>
+            <article class="container footer-bottom">
+                <div class="footer-logo">
+                    
+                </div>
+                <div class="footer-description">
+                    <p>
+                        When we started BKWebDev, our goal was to give anyone in the world 
+                        the ability to learn the skills they’d need to succeed in the 21st century. 
+                        We set out to create a new, interactive way of learning — making it engaging, 
+                        flexible, and accessible for as many people as possible. 
+                        Since then, we have helped millions of people worldwide unlock 
+                        modern technical skills and reach their full potential through code. ✨
+                    </p>
+                    <small>Made with ❤️in NYC © 2023 BWD</small>
+                   
+                </div>
+            </article>
```

Comment:
- You have not styled this section of your project correctly, please note that in the mobile version, the footer has a dark background color, while in the desktop version, the footer has a white background color. Please fix this 🙏
Your project:
Mobile version:
<img width="345" alt="ewefwefwef" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/1c09bcf3-56dc-4a8a-ab27-1a801b66a9ec">
Desktop version:
<img width="512" alt="eruerer" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/bd3ee7fa-3fbc-4d9c-8fd7-7f1ade7ecf48">

Design template:
Mobile version:
<img width="304" alt="gergerv" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/0a650689-ee01-455b-997e-ff1c5e18bc51">
Desktop version:
<img width="556" alt="gwfwg" src="https://github.com/Bettenoch/Web_Development_Course/assets/73607512/66063a65-8c95-4db1-994a-cab8be3ac6a5">